py/desispec/qa/qa_plots.py

```diff
@@ -156,30 +159,99 @@ def frame_skyres(outfil, frame, skymodel, qaframe):
 
     # Histogram of all residuals
     ax1 = plt.subplot(gs[1,0])
-    binsz = 0.1
     xmin,xmax = -5., 5.
-    i0, i1 = int( np.min(devs) / binsz) - 1, int( np.max(devs) / binsz) + 1
-    rng = tuple( binsz*np.array([i0,i1]) )
-    nbin = i1-i0
+
+    ##binsz = 0.1
+    ##i0, i1 = int( np.min(devs) / binsz) - 1, int( np.max(devs) / binsz) + 1
+    ##rng = tuple( binsz*np.array([i0,i1]) )
+    ##nbin = i1-i0
     # Histogram
-    hist, edges = np.histogram(devs, range=rng, bins=nbin)
+    ##hist, edges = np.histogram(devs, range=rng, bins=nbin)
+
+    binsz = qaframe.qa_data['SKYSUB']["PARAMS"]["BIN_SZ"]
+    hist = np.asarray(qaframe.qa_data['SKYSUB']["METRICS"]["DEVS_1D"])
```

Comment:
Initially what was there was a sky residual calculation repeated in
   the plotting routine itself, even though it was already done in the sky
   resid function.  In addition, the hist and edges quantities are metrics
   of the physical distribution of residuals like other QAs.  This
   calculation is also something we would want in quicklook, for instance.
   By refactoring, we omit the duplicate calculations, and modularize the
   code so that sharing offline and online is straightforwad, which is a
   main part of the purpose of this PR.  The offline plotting
   functionality of residuals is preserved.